bk-config/bin/create_known_hosts_file.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+
+if [ ! -e /usr/lib/sage/certca/sage_beekeeper_ca.pub ] ; then
+    echo CA public key not found!
+    exit 1
+fi 
+
+
+if [ $# -eq 0 ]
+  then
+    echo "Usage: create_known_hosts_file.sh HOST PORT"
+    echo ""
```

Comment:
i would use the same language as in create_client_files.sh for what these parameters mean

HOST: the name of the beekeeper host the client is going to connect to
PORT: the port of the above mentioned host